Documentation/ApiOverview/Context/_MyControllerUsingDependencyInjection.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Controller;
+
+use TYPO3\CMS\Core\Context\Context;
+
+final class MyController
+{
+    /**
+     * @var Context
```

Comment:
docblock is redundant to the type declaration and can be omitted.

Reply:
What is a "docblock"?

Reply:
The comment above the type definition.

Reply:
I have removed the docblock now.